core/plugins/org.polarsys.capella.core.platform.eclipse.ui.trace/plugin.properties

```diff
@@ -1,15 +1,16 @@
-# =============================================================================
-# Copyright (c) 2006, 2015 THALES GLOBAL SERVICES.
-# 
-# This program and the accompanying materials are made available under the
-# terms of the Eclipse Public License 2.0 which is available at
-# http://www.eclipse.org/legal/epl-2.0
-# 
-# SPDX-License-Identifier: EPL-2.0
-# 
-# Contributors:
-#    Thales - initial API and implementation
-# =============================================================================
-#Properties file for org.polarsys.capella.core.platform.eclipse.ui.trace
-providerName = Eclipse.org
-pluginName = MDTrace Eclipse UI For Capella
\ No newline at end of file
+# =============================================================================
+# Copyright (c) 2006, 2015 THALES GLOBAL SERVICES.
+# 

```

Comment:
update header, change the last modification from 2015 to 2020.

Copyright (c) 2006, 2020 THALES GLOBAL SERVICES. ....